pkg/eventmanager/eventmanager.go

```diff
@@ -29,12 +29,14 @@ const (
 	UPGRADE_EXTDEPCHECK_DELAY_DESC = "Cluster upgrade to version %s is experiencing a delay as an external dependency of the upgrade is currently unavailable. The upgrade will continue to retry. This is an informational notification and no action is required by you."
 	// UPGRADE_SCALE_DELAY_DESC describes the upgrade scaling delayed
 	UPGRADE_SCALE_DELAY_DESC = "Cluster upgrade to version %s is experiencing a delay attempting to scale up an additional worker node. The upgrade will continue to retry. This is an informational notification and no action is required by you."
+	// UPGRADE_SCALE_DELAY_SKIP_DESC describes the upgrade scaling skipped after delay
+	UPGRADE_SCALE_DELAY_SKIP_DESC = "Cluster upgrade to version %s has experienced issue during capacity reservation efforts. This could be caused by the AWS account service quota limitation, or the temporary connectivity issue from the new worker node to the masters or the external services. The upgrade will continue without extra compute. This is an informational notification and no action is required by you."
```

Comment:
Made some slight tweaks - a couple of tiny grammar things and just simplifying the issues around the worker.

```suggestion
	UPGRADE_SCALE_DELAY_SKIP_DESC = "Cluster upgrade to version %s has experienced an issue during capacity reservation efforts. This could be caused by AWS account service quota limitations or temporary connectivity issues to/from the new worker node. The upgrade will continue without extra compute. This is an informational notification and no action is required by you."
```